packages/host/app/components/operator-mode/container.gts

```diff
@@ -840,6 +844,7 @@ export default class OperatorModeContainer extends Component<Signature> {
       }
 
       .container__chat-sidebar {
+        height: 100vh;
```

Comment:
There may be a better solution but on main opening a larger chat stretches everything off of the page without it.